setup.py

```diff
@@ -67,6 +68,9 @@
                 'apps/morph_check',
                 'apps/morph_stats',
                 ],
+    'entry_points':{
+        'console_scripts': ['neurom=apps.__main__:cli']
```

Comment:
This doesn't seem to work in an editable install; could be b/c of __init__.py not being there, but that will probably trigger other problems:

```
Traceback (most recent call last):
  File "/home/asdf/src/NeuroM/venv/bin/neurom", line 11, in <module>
suggestion
        'console_scripts': ['neurom=apps.__main__:cli']

    load_entry_point('neurom', 'console_scripts', 'neurom')()
  File "/home/asdf/src/NeuroM/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 542, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/asdf/src/NeuroM/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2569, in load_entry_point
    return ep.load()
  File "/home/asdf/src/NeuroM/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2229, in load
    return self.resolve()
  File "/home/asdf/src/NeuroM/venv/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2235, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ImportError: No module named apps.__main__
```

Reply:
I have added the \_\_init\_\_.py file

Reply:
Does it work for you know (it was already working on my side)

Reply:
yup, seems to